flow/scheduling/base.py

```diff
@@ -45,12 +49,47 @@ class JobStatus(enum.IntEnum):
     error = 8
     """The cluster job is in an error or failed state."""
 
+    group_registered = 9
+    """The operation via a group is registered with the scheduler.but no other status is known."""
```

Comment:
What does "via a group" mean? I would prefer phrasing like:
```suggestion
    """The operation is in a group that is registered with the scheduler but no other status is known."""
```